assets/js/googlesitekit/modules/create-info-store.js

```diff
@@ -53,13 +53,40 @@ export const createInfoStore = (
 ) => {
 	invariant( storeName, 'storeName is required.' );
 
+	const WAIT_FOR_REAUTH_RESOLVERS = 'WAIT_FOR_REAUTH_RESOLVERS';
```

Comment:
This seems fine but it's worth noting that similar constants are defined at the top level in the other `create.*store` modules, so if we want to be entirely consistent we could move it up a level.

E.g.:
https://github.com/google/site-kit-wp/blob/fe3f0b4f1c53f1dd422a43db2b1fdc0afdba80be/assets/js/googlesitekit/modules/create-submit-changes-store.js#L31-L36